components/ui/markdown/RenderMarkdown.tsx

```diff
@@ -46,7 +46,7 @@ const RenderMarkdown = ({
   const HEADING_4_SIZE = isSystemTextSize ? 18 / fontScale : 15 + fontModifier;
 
   const onLinkPress = (url): boolean => {
-    openLink(url, navigation);
+    openLink(url, navigation, currentAccount.instance);
```

Comment:
Don't need to pass in the instance anymore.

Reply:
Fixed in the latest commit. Thanks!